src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -650,6 +652,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
       {errors.iframeUrl && (
         <span className="input-error">{errors.iframeUrl.message}</span>
       )}
+      <label
+        htmlFor={"autoPlaycheckbox"}
+        className={`checkbox ${
+          !watch("autoPlay", false) && "checkbox-checked"
+        }`}
+      >
+        If you are embedding a video, do you want to disable autoplay?
```

Comment:
Is it going to be a checkmark?
Maybe not present it as an opt-in or opt-out but instead:

Autoplay your embeded video ☑️

Reply:
fixed in 418cdcb4763615a45870bc6b1dfd26c2ec198a84